src/Tests/runTest.ts

```diff
@@ -43,11 +43,25 @@
 
 import {runTests} from '@vscode/test-electron';
 import * as path from 'path';
+import {argv} from 'process';
 
 async function main() {
+  let ci: boolean = false;
+  if (argv.length > 2) {
+    // front of two args
+    let args = argv.splice(2);
+    args.forEach((str) => {
+      if (str === 'ci') {
+        ci = true;
+      }
+    });
+  }
```

Comment:
It can be reduced as following codes I think :)

```suggestion
  let ci: boolean = argv.length > 2 && argv.slice(2).includes('ci');
```

Reply:
🤑